### PR TITLE
Replace custom tooltips with native tooltips in personal and administration settings

### LINKED ISSUE #35886: [BITV]: Replace custom tooltips with native one for personal and administration settings
## Problem

To show a tooltip on a several components we currently use v-tooltip. An example can be found in https://github.com/nextcloud/server/blob/master/apps/settings/src/components/PersonalInfo/shared/FederationControl.vue#L37

v-tooltip is not accessible and we want to migrate to native tooltips (title attribute) instead. An overview can be found in https://github.com/nextcloud/nextcloud-vue/issues/2503.

## Solution

Add a `title` prop to several components that will render as `title` attribute on the HTML element.

### PR DESCRIPTION
**Pull request by JuliaKirschenheuter**
* Resolves: https://github.com/nextcloud/server/issues/35886

## Summary

Replace custom tooltips with native tooltips in personal and administration settings sections besides "Mobile & desktop", "Privacy" and "System"

## Checklist

- Code is [properly formatted](https://docs.nextcloud.com/server/latest/developer_manual/digging_deeper/continuous_integration.html#linting)
- [Sign-off message](https://github.com/src-d/guide/blob/master/developer-community/fix-DCO.md) is added to all commits
- [ ] Tests ([unit](https://docs.nextcloud.com/server/latest/developer_manual/app_development/tutorial.html#unit-tests), [integration](https://docs.nextcloud.com/server/latest/developer_manual/app_development/tutorial.html#integration-tests), api and/or acceptance) are included
- [ ] Screenshots before/after for front-end changes
- [x] Documentation ([manuals](https://github.com/nextcloud/documentation/) or wiki) has been updated or is not required
- [x] [Backports requested](https://github.com/nextcloud/backportbot/#usage) where applicable (ex: critical bugfixes)

### CODE REVIEW

File: apps/settings/src/components/AppDetails.vue [hunk 1]
```diff
@@ -121,7 +123,7 @@
 				class="appslink"
 				:href="app.bugs"
 				target="_blank"
-				rel="noreferrer noopener">{{ t('settings', 'Report a bug') }} ↗</a>
+				rel="noreferrer noopener">{{ t('settings', 'Visit websitg') }} ↗</a>
```
Comment (ChristophWurst) [inline apps/settings/src/components/AppDetails.vue:126]: typo

**Review by ChristophWurst — CHANGES_REQUESTED** [review]

**Review by ChristophWurst — APPROVED** [review]
The disappearance of ``dist/icons.css`` is suspicious once again

**Review by artonge — APPROVED** [review]

**Review by Pytal — APPROVED** [review]

**Comment by PVince81** [comment]
@JuliaKirschenheuter  please check handlebars failure

**Comment by ChristophWurst** [comment]
Time for https://docs.nextcloud.com/server/latest/developer_manual/getting_started/development_process.html#manual-backport

**Comment by nickvergessen** [comment]
> - [ ] Screenshots before/after for front-end changes

Wondering why there are frontend changes without screenshots. The template should help to remind you about them.

Here are the screenshots:
Before | After
---|---
Bildschirmfoto vom 2023-01-17 22-08-38 | Bildschirmfoto vom 2023-01-17 22-09-07

Can you fix that in a follow up? Thanks

**Comment by JuliaKirschenheuter** [comment]
> Can you fix that in a follow up? Thanks

Thank you! Of coarse i will fix it. As a follow-up

**Pull request merged**